doc/spinel-protocol-src/spinel-commands.md

```diff
@@ -219,3 +219,84 @@ See (#security-considerations) for more information.
 
 This command requires the capability `CAP_PEEK_POKE` to be present.
 
+## CMD 21: (Host->NCP) CMD_PROP_VALUE_MULTI_GET {#prop-value-multi-get}
+
+*   Argument-Encoding: `A(i)`
+*   Required Capability: `CAP_CMD_MULTI`
+
+Fetch the value of multiple properties in one command. Arguments are
+an array of property IDs. If all properties are fetched successfully,
+a `CMD_PROP_VALUES_ARE` command is sent back to the host containing
+the propertyid and value of each fetched property. The order of the
+results in `CMD_PROP_VALUES_ARE` match the order of properties given
+in `CMD_PROP_VALUE_GET`.
+
+Errors fetching individual properties are reflected as indicating a
+change to `PROP_LAST_STATUS` for that property's place.
+
+Not all properties can be fetched using this method. As a general rule
+of thumb, any property that blocks when getting will fail for that
+individual property with `STATUS_INVALID_COMMAND_FOR_PROP`.
+
+## CMD 22: (Host->NCP) CMD_PROP_VALUE_MULTI_SET {#prop-value-multi-set}
+
+*   Argument-Encoding: `A(T(iD))`
+*   Required Capability: `CAP_CMD_MULTI`
+
+Octets: |    1   |          1               |   *n*
+--------|--------|--------------------------|----------------------
+Fields: | HEADER | CMD_PROP_VALUE_MULTI_SET | Property/Value Pairs
+
+With each property/value pair being:
+
+Octets: |    2   |   1-3   |  *n*
+--------|--------|---------|------------
+Fields: | LENGTH | PROP_ID | PROP_VALUE
+
+This command sets the value of several properties at once in the given
+order. The setting of properties stops at the first error, ignoring
+any later properties.
+
+The result of this command is generally `CMD_PROP_VALUES_ARE` unless
+(for example) a parsing error has occured (in which case
+`CMD_PROP_VALUE_IS` for `PROP_LAST_STATUS` would be the result). The
```

Comment:
What should be the behavior for the case where some "property set" in middle of list has a parsing error? I see two options:

1. We reply with `CMD_PROP_VALUES_ARE`  and send `LAST_STATUS` `PARSE_ERROR` for the failed "set property" in the list. 
2. Send a `VALUE_IS` with a single `PARSE_ERROR` in response.

I prefer the first option.